flows/vision-zero/prefect_agent_environment_variables.env_template

```diff
@@ -9,12 +9,36 @@ AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 AWS_CSV_ARCHIVE_BUCKET_NAME=
 AWS_CSV_ARCHIVE_PATH_PRODUCTION=
+#AWS_CSV_ARCHIVE_PATH_STAGING=
 AWS_CSV_ARCHIVE_PATH_STAGING=
+
+DB_HOST=
+DB_USER=
+DB_PASS=
+DB_NAME=
+DB_IMPORT_SCHEMA=
+GRAPHQL_ENDPOINT=
+GRAPHQL_ENDPOINT_KEY=
+
 PRODUCTION_GRAPHQL_ENDPOINT=
 PRODUCTION_GRAPHQL_ENDPOINT_ACCESS_KEY=
 OCR_CR3_SOURCE_BUCKET=
 OCR_CR3_SOURCE_PATH=
 OCR_DIAGRAM_TARGET_BUCKET=
 OCR_DIAGRAM_TARGET_PATH=
-OCR_BATCH_SIZE=
+OCR_BATCH_SIZE=10
 OCR_SINGLE_CRASH=
+
+AFD_DB_USERNAME=
+AFD_DB_PASSWORD=
+AFD_DB_HOSTNAME=
+AFD_DB_PORT=5432
+AFD_DB_DATABASE=
```

Comment:
I am inclined to be totally fine with defining the database connection parameters again here, even though it's logical to assume that they will always match `DB_HOST`, `DB_USER`, etc as used in different flow in this project. Environment variables / flow parameters have no cost or performance impact, and I think how much they are isolated or sharing configuration variables with related flows can be assessed on a case by case basis by the developer.